vision_evaluation/evaluators.py

```diff
@@ -1044,5 +1175,73 @@ def add_predictions(self, predictions, targets):
         for pred_mask, gt_mask in zip(predictions, targets):
             pred_mask = np.asarray(pred_mask)
             gt_mask = np.asarray(gt_mask)
-            mean_l1 = np.abs(pred_mask.astype(np.float)-gt_mask.astype(np.float)).mean()
+            mean_l1 = np.abs(
+                pred_mask.astype(np.float) - gt_mask.astype(np.float)).mean()
             self._metric_sum += mean_l1
+
+
+class GroupWiseEvaluator(Evaluator):
+    """
+    Group Wise Evaluator
+    """
+    def __init__(self, evaluator, task_name, group_name,
+                 multitask_manifest_dataset, target_task_predictions):
+        """
+        Args:
+            evaluator (vision_evaluation.Evaluator): Evaluation class object. E.g. TopAccuracyEvaluator()
+            
+            manifest_dataset (ManifestDataset): Iterable multitask ManifestDataset object.
+            
+            target_task_predictions (N, num_class):  Model output numpy array for the target task.
```

Comment:
The prediction results has to be given to the constructor? add_prediction() method is not used?